.github/workflows/gosec.yml

```diff
@@ -0,0 +1,26 @@
+name: Build CI
```

Comment:
Can we change the name to something like `Check Gosec Security Scanner`? Helps with better CI/CD naming to in GH output in PRs.

Reply:
Done.